docs/content/latest/explore/ysql-language-features/tablespaces.md

```diff
@@ -0,0 +1,156 @@
+
+DEV INPUT REQUIRED
+
+```sql
+CREATE TABLESPACE us_west_1a_tablespace 
+WITH (replica_placement='{"num_replicas": 1, "placement_blocks":
+[{"cloud":"aws","region":"us-west","zone":"us-west-1a","min_num_replicas":1}]}');
+```
+
+If you create a table and use the preceding tablespace, each tablet of the table will have one replica, as per the `num_replicas` value. Since `placement_blocks` contains one placement block where `min_num_replicas` is set to 1, YB-Load balancer will ensure that `aws.us-west.us-west-1a` will have one replica.
+
+## Creating Tables and Indexes in Tablespaces
+
+You can associate new tables and indexes with a corresponding tablespace. This defines the replication factor of the table or index. It also defines how the replicas of the table or index are to be spread across cloud, regions, and zones.
+
+### How to Use a Multi-Zone Tablespace
+
+Using the multi-zone tablespaces defined in [How to Create a Multi-Zone Tablespace](how-to-create-a-multi-zone-tablespace), you can apply the YSQL's `TABLESPACE` option to `CREATE TABLE` and `CREATE INDEX` statements, as follows:
+
+```sql
+CREATE TABLE employees (
+  employee_no integer PRIMARY KEY,
+  name text,
+  department text,
+  change_date date
+)
+TABLESPACE us_west_tablespace;
+```
+
+```sql
+CREATE INDEX employee_no_idx ON employees(employee_no) 
+TABLESPACE us_east_tablespace;
+```
+
+The preceding statements ensure that data in the `employees` and `employee_no_idx` tables is present based on the placement policies specified for `us_west_tablespace` and `us_east_tablespace` respectively.
+
+### How to Use a Single-Zone Tablespace
+
+Using the single-zone tablespace defined in [How to Create a Single-Zone Tablespace](how-to-create-a-single-zone-tablespace), you can apply the YSQL's `TABLESPACE` option to `CREATE TABLE` and `CREATE INDEX` statements, as follows:
+
+DEV INPUT REQUIRED
+
+```sql
+CREATE TABLE employees (
+  employee_no integer PRIMARY KEY,
+  name text,
+  department text,
+  change_date date
+)
+TABLESPACE us_west_1a_tablespace;
+```
+
+```sql
+CREATE INDEX employee_no_idx ON employees(employee_no) 
+TABLESPACE us_east_1a_tablespace;
+```
+
+The preceding statements ensure that data in the `employees` and `employee_no_idx` tables is present based on the placement policies specified for `us_west_1a_tablespace` and `us_east_1a_tablespace` respectively.
+
+## Dropping Tablespaces
+
+You can drop a tablespace in YSQL using the following syntax:
+
+```sql
+DROP TABLESPACE tablespace_name;
+```
+
+The preceding operation cannot be performed if the tablespace has associated tables or indexes.
+
+## Limitations
+
+The current release of YugabyteDB has the following limitations related to tablespaces:
+
+1. `ALTER TABLE` in combination with `SET TABLESPACE` is not supported.
+2. `ALTER TABLESPACE` is not supported.
+3. Setting read replica placements and affinitized leaders using tablespaces is not supported.
+4. Setting tablespace for a database is not supported.
```

Comment:
5. Setting tablespace for colocated tables/databases is not yet supported

Reply:
Added this.